### DISCUSSION TITLE
The picture was uploaded incorrectly

### DISCUSSION THREAD

**Discussion by JokerPan12** [misskey-dev/misskey#14618] [Q&A] [answered]
### 💡 Summary

<img width="492" alt="image" src="https://github.com/user-attachments/assets/9bede5a0-9b79-4fae-9baa-8da95d2da3bd">

When uploading images using Docker deployment, the above error occurs.

### 🥰 Expected Behavior

The uploading of images should be fine.

### 🤬 Actual Behavior

The uploading of images should be fine.

### 💻 Frontend Environment

```markdown
* Model and OS of the device(s):
* Browser:
* Server URL:
* Misskey:
```

### 🛰 Backend Environment (for server admin)

```markdown
* Installation Method or Hosting Service:
* Misskey:
* Node:
* PostgreSQL:
* Redis:
* OS and Architecture:
```

### Do you want to address this bug yourself?

- [ ] Yes, I will patch the bug myself and send a pull request

**Comment by JokerPan12** [comment]
<img width="809" alt="image" src="https://github.com/user-attachments/assets/f83b888e-ebb6-4956-b81b-3b06b472f6de">

**Reply by KisaragiEffective** [reply]
It does not mean other than there is internal error, literally.
Please provide information about environment to identify cause by eagerly someone.

**Reply by JokerPan12** [reply]
After deploying with docker, an error occurred when uploading pictures
The following is container information
<img width="838" alt="image" src="https://github.com/user-attachments/assets/5682ca0a-ad43-47fa-b1dc-867108391304">

Mount/Map：
<img width="698" alt="image" src="https://github.com/user-attachments/assets/65908677-7012-4e93-8100-8b097d1f8b48">

/www/dk_project/dk_app/dk_misskey/config   ---   /misskey/.config
/www/dk_project/dk_app/dk_misskey/files    ---     /misskey/files

**Reply by JokerPan12** [reply]
> It does not mean other than there is internal error, literally. Please provide information about environment to identify cause by eagerly someone.

The following is the container log
<img width="687" alt="image" src="https://github.com/user-attachments/assets/36dbf785-c7c5-4c60-a782-61baae605ad6">

**Reply by chocolate-pie** [reply, marked as answer]
Misskey needs write permission to destination folder (e.g. /misskey/files). I don’t know chinese, but I seem you need to change ‘权限’ to something else.

**Reply by JokerPan12** [reply]
> Misskey needs write permission to destination folder (e.g. /misskey/files). I don’t know chinese, but I seem you need to change ‘权限’ to something else.

The problem has been resolved. Thank You!
